ext/session/session.c

```diff
@@ -485,19 +485,24 @@ PHPAPI int php_session_valid_key(const char *key) /* {{{ */
 /* }}} */
 
 
-static void php_session_gc(void) /* {{{ */
+static zend_long php_session_gc(zend_bool immediate) /* {{{ */
 {
```

Comment:
I agree casting  is ugly. I don't mind much about zend_long or int because there wouldn't be int overflow by session_gc().